public/pages/Indices/containers/Indices/Indices.tsx

```diff
@@ -37,6 +40,7 @@ import { SECURITY_EXCEPTION_PREFIX } from "../../../../../server/utils/constants
 import IndicesActions from "../IndicesActions";
 import { destroyListener, EVENT_MAP, listenEvent } from "../../../../JobHandler";
 import "./index.scss";
+import IndexDetail from "../../../IndexDetail/containers/IndexDetail";
```

Comment:
```suggestion
import IndexDetail from "../../containers/IndexDetail";
```
You are using wrong component here.

Reply:
The IndexDetail inside IndexDetail page is a super heavy component and should not be rendered inside a cell of data grid.